impl/client/src/main/javascript/web/pentaho/i18n.js

```diff
@@ -104,11 +107,14 @@ define([
 
     var absBundleUrl = localRequire.toUrl(bundleMid);
 
-    // Remove basePath from bundle url
     var serverUrl = context.server.url;
-    var basePath = (serverUrl && serverUrl.pathname) || "";
-    if(basePath && absBundleUrl.indexOf(basePath) === 0)
-      absBundleUrl = absBundleUrl.substr(basePath.length);
+
+    // Remove basePath from bundle url
+    // Taking into account embedded scenarios, where it will be a full URL - "http://host:port/..."
+    var reminderUrlRegX = new RegExp("^(?:" + serverUrl + "|" + serverUrl.origin + "|" + serverUrl.pathname + ")(.*)");
+    var reminderUrlMatch = reminderUrlRegX.exec(absBundleUrl);
```

Comment:
This two variables should be 'remainderUrlRegX' and 'remainderUrlMatch' respectively